### ISSUE TITLE
calendar heatmap - scroll doesn't work

### ISSUE THREAD

**Issue by metasyn** [hobbes3/dos_viz#2]
left works the first time; then it breaks. right never works

**Comment by hobbes3** [comment]
Yup, I've noticed that as well. The original view from Web Framework Toolkit also exhibit this behavior.
